### PR TITLE
Fix ViewSuiBridge Bridge CLI command

### PR DESCRIPTION
**Pull request by Bridgerz**
## Description 

Handle the case where a bridge committee member is no longer a validator. 

## Test plan 

Tested locally.

### CODE REVIEW

File: crates/sui-bridge-cli/src/main.rs [hunk 1]
```diff
@@ -410,21 +410,21 @@ async fn main() -> anyhow::Result<()> {
...
-                if ping {
-                    let client_clone = client.clone();
-                    ping_tasks.push(client_clone.get(url.clone()).send());
+                if let Some(name) = names.get(&sui_address) {
+                    if ping {
+                        let client_clone = client.clone();
+                        ping_tasks.push(client_clone.get(url.clone()).send());
+                    }
+                    authorities.push((
+                        name,
```
[the remark this answers is not in the document]
Comment (Bridgerz) [reply crates/sui-bridge-cli/src/main.rs:419]: wym

[the remark this answers is not in the document]
Comment (longbowlu) [reply crates/sui-bridge-cli/src/main.rs:419]: if it's not in `names`, use `url` as the name, so at least we know who they are

[the remark this answers is not in the document]
Comment (Bridgerz) [reply crates/sui-bridge-cli/src/main.rs:419]: oh thats a great idea

[the remark this answers is not in the document]
Comment (Bridgerz) [reply crates/sui-bridge-cli/src/main.rs]: Fixed!

**Review by longbowlu — APPROVED** [review]

**Pull request merged**